docs/api.md

```diff
@@ -1126,3 +1126,86 @@ HTTP Status Code 200
     "ok": true
 }
 ```
+
+### Get Teacher Prices
+
+```
+GET /api/v1/step_prices/
```

Comment:
You may use `/api/v1/teacher/1/school/1/prices` as the url so that no need to add parameters.